nnvm/tests/python/frontend/darknet/test_forward.py

```diff
@@ -441,7 +462,8 @@ def test_forward_activation_logistic():
     test_forward_resnet50()
     test_forward_alexnet()
     test_forward_extraction()
-    test_forward_yolo()
+    test_forward_yolov2()
+    test_forward_yolov3()
```

Comment:
yolo and region have multiple outputs from tvm, all the outputs are not validated

Reply:
Done